packages/api/.eslintrc.json

```diff
@@ -1,4 +1,7 @@
 {
     "extends": "@comet/eslint-config/nestjs",
-    "ignorePatterns": ["lib/**"]
+    "ignorePatterns": ["lib/**"],
+    "rules": {
+        "@comet/no-other-module-relative-import": "off"
+    }
```

Comment:
why is this off?

i think we should enable that, maybe in an extra PR if it is a lot work.

Reply:
I copied this from the comet repo where we disabled it for all packages (see https://github.com/vivid-planet/comet/pull/1303/commits/2164c97620addd8d40f97d87c593427629b03cf7).

But I'm not sure what the reason was. If I remember correctly, there was some problem with setting up the `@src` alias in a package. Maybe @johnnyomair can comment?

Reply:
We can't have an import alias in the packages because that's a feature from transpilers such as Babel. TSC doesn't support it directly, and we decided to not use a workaround that patches it to achieve the import alias.